src/screens/OrganizationDashboard/OrganizationDashboardMocks.ts

```diff
@@ -298,3 +299,23 @@ export const ERROR_MOCKS = [
     error: new Error('Mock Graphql ORGANIZATION_EVENT_LIST Error'),
   },
 ];
+
+export const mockEventData = {
+  eventsByOrganizationConnection: [
+    { startDate: new Date().toISOString() }, // Assuming an event is upcoming
+  ],
+};
+
+// Mocking the response for the organization event query
+export const linkMocked = new StaticMockLink(
+  [
+    {
+      request: {
+        query: ORGANIZATION_EVENT_CONNECTION_LIST,
+        variables: { organization_id: 'your_organization_id' },
+      },
+      result: { data: mockEventData },
+    },
+  ],
+  true
+);
```

Comment:
No need to create `linkMocked` here, move 
```ts
    {
      request: {
        query: ORGANIZATION_EVENT_CONNECTION_LIST,
        variables: { organization_id: 'your_organization_id' },
      },
      result: { data: mockEventData },
    },
```
to `MOCKS`. Please follow the convention.

Reply:
Use `link1` then.

Reply:
I insert this code into the  `MOCKS`
  
```
    {
      request: {
        query: ORGANIZATION_EVENT_CONNECTION_LIST,
        variables: { organization_id: 'your_organization_id' },
      },
      result: { data: mockEventData },
    },
```
Please Check, is it right or wrong